src/game_battlealgorithm.cpp

```diff
@@ -504,21 +504,21 @@ Game_Battler* Game_BattleAlgorithm::AlgorithmBase::GetTarget() const {
 }
 
 float Game_BattleAlgorithm::AlgorithmBase::GetAttributeMultiplier(const std::vector<bool>& attributes_set) const {
-	float multiplier = 0;
-	int attributes_applied = 0;
+	float physical = -1001, magical = -1001;
```

Comment:
> "3.16: RM2000: Attribute modifier is the maximum of the percentage da… 

The actual minimum in the editor (for E) is -9999, so this values should be at least -10000. (that you can set E to something negative is btw also totally non-obvious but documented in the help file)

Reply:
Because #1394 copy-pastes this function into Game_Battler I suggest to remove it from here (or make it a thin wrapper)and just call the Game_Battler function. I will clean it up in the other PR afterwards.